### ISSUE TITLE
Document that intermediate definitions are okay

### ISSUE THREAD

**Issue by mgold** [stoeffel/elm-verify-examples#31]
The README doesn't document a useful feature: If I define values, they will be included with the test. Importantly this only works when those values have a type annotation; otherwise it's a compiler error.

Example:
```elm
    findMe : Int
    findMe = 42

    minimizeMe : Int -> Int
    minimizeMe n = abs (n-findMe)

    nearest : Int
    nearest = foldl1 (\a b -> if minimizeMe a < minimizeMe b then a else b) (Nonempty 10 [20,30,40,50,60])

    nearest --> 40
```

**Comment by stoeffel** [comment]
yes, this needs docs. also help with the docs would be highly appreciated.

**Comment by stoeffel** [comment]
Documented and fixed a bug related to this feature.
Published in v1.0.1.

Thanks for issuing ❤️ Hope this addresses this issue.
